recursive_count_th/count_th.py

```diff
@@ -3,8 +3,13 @@
 Your function should return a count of how many occurences of ***"th"*** occur within `word`. Case matters.
 Your function must utilize recursion. It cannot contain any loops.
 '''
-def count_th(word):
+def count_th(word, count = 0):
+    if len(word) < 2: #base case 
+        return count
     
-    # TBC
-    
-    pass
+    #if th in word we slice from 2nd index, else from 1st index
+    if word[:2] == 'th':
+        count += 1
+        return count_th(word[2:], count)
+    else:   
+        return count_th(word[1:], count)
```

Comment:
Nice job with this one as well!

You're technically not supposed to modify the signature of the method, but you're a few trivial modifications away from arriving at a solution that doesn't modify the signature. So, I'm counting this as correct.

Just be careful when reading the instructions on future Sprint Challenges. But, nice job with this.